src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py

```diff
@@ -669,12 +670,18 @@
     text: az postgres flexible-server replica list -g testGroup -n testserver
 """
 
-helps['postgres flexible-server replica stop-replication'] = """
+helps['postgres flexible-server replica promote'] = """
```

Comment:
i hope we are not removing support for stop-replication operation completely as this will be considered as breaking change. Not sure if that will be allowed